rootfs/api/models/app.py

```diff
@@ -494,7 +494,7 @@ def deploy(self, release, force_deploy=False, rollback_on_failure=True):  # noqa
             raise DeisException('No build associated with this release')
 
         app_settings = self.appsettings_set.latest()
-        addresses = ",".join(address for address in app_settings.whitelist)
+        addresses = ",".join(address for address in app_settings.whitelist) if app_settings.whitelist else None  # noqa
```

Comment:
I think it would be more Pythonic to spell this one out instead of make it a one-liner that needs `# noqa`:
```python
if app_settings.whitelist:
    addresses = ",".join(address for address in app_settings.whitelist)
else:
    addresses = None
```